### PR TITLE
Adding gemvBatched and gemvStridedBatched with cuda backend

### PR DESCRIPTION
**Pull request by daineAMD**
This adds hipblasXgemvBatched and hipblasXgemvStridedBatched when using cuBLAS 11.7.0 or greater as a backend.

### CODE REVIEW

File: clients/include/blas2/testing_gemv_batched.hpp [hunk 1]
```diff
@@ -311,6 +316,7 @@ void testing_gemv_batched(const Arguments& arg)
                                                       batch_count);
         EXPECT_HIPBLAS_STATUS(
             actual, (invalid_size ? HIPBLAS_STATUS_INVALID_VALUE : HIPBLAS_STATUS_SUCCESS));
+	*/
         return;
     }
```
Comment (TorreZuk) [inline clients/include/blas2/testing_gemv_batched.hpp:324]: Why not test with guard?   I would put in a TODO, but if you don't want double coverage put in a SKIPPED so these can be removed when seen in test run.

File: library/src/nvidia_detail/hipblas.cpp [hunk 2]
```diff
@@ -4847,8 +4876,29 @@ hipblasStatus_t hipblasZgemvBatched(hipblasHandle_t                   handle,
                                     hipblasDoubleComplex* const       y[],
                                     int                               incy,
                                     int                               batchCount)
+try
 {
+#if CUBLAS_VERSION >= 110700
+    return hipCUBLASStatusToHIPStatus(cublasZgemvBatched((cublasHandle_t)handle,
```
Comment (TorreZuk) [inline library/src/nvidia_detail/hipblas.cpp:4882]: Probably should make a future TODO ticket to review moving to a getProcAddress query system so we can cover a few major releases with the same build.   These are for the C API so shouldn't be too tricky.

**Review by TorreZuk — APPROVED** [review]
LGTM future work suggestions

File: library/src/nvidia_detail/hipblas.cpp (hunk 2 above)
Comment (daineAMD) [reply library/src/nvidia_detail/hipblas.cpp:4882]: Created [LWPMLSE-523](https://ontrack-internal.amd.com/browse/LWPMLSE-523)

**Pull request merged**